portal-impl/src/com/liferay/portal/util/PortalImpl.java

```diff
@@ -2880,7 +2693,7 @@ public String getJsSafePortletId(String portletId) {
 
 	@Override
 	public String getLayoutActualURL(Layout layout) {
-		return getLayoutActualURL(layout, getPathMain());
+		return _getLayoutActualURL(layout, getPathMain());
```

Comment:
@slnn You replace these calls with a private method, but I don't see where your logic comes from. Did you also plan to remove `public String getLayoutActualURL(Layout layout, String mainPath)`?

Reply:
Oh, my bad, I planned to remove public String getLayoutActualURL(Layout layout, String mainPath), but I found it is not unused, however, I forget to remove the temporary change,  I will double check and force push.

Reply:
Oh, my bad, I planned to remove public String getLayoutActualURL(Layout layout, String mainPath), but I found it is not unused, however, I forget to remove the temporary change,  I will double check and force push.